docs/contents/user-docs/authorization-and-access.md

```diff
@@ -0,0 +1,69 @@
+# Configure permissions for authorization and access
```

Comment:
Would be great to include an overview description of what this document achieves? Not particularly obvious from the headers.

Reply:
@RedbackThomson https://aws-controllers-k8s.github.io/community/user-docs/authorization/